packages/gatsby-www-components/src/components/horizontal-nav-list.js

```diff
@@ -0,0 +1,40 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { Link } from "gatsby"
+
+import { mediaQueries } from "gatsby-design-tokens"
+
+const HorizontalNavList = ({ items = [], slug }) => (
+  <nav>
+    <ul
+      sx={{
+        m: 0,
+        display: `flex`,
+        flexWrap: `wrap`,
+        listStyle: `none`,
+        "& * + *": {
+          borderLeftStyle: `solid`,
+          borderLeftWidth: `1px`,
+          borderColor: `ui.border`,
+        },
+      }}
+    >
+      {items.map(item => (
+        <li
+          sx={{
+            m: 0,
+            padding: `3px 12px`,
+            [mediaQueries.md]: {
+              padding: `0px 6px`,
+            },
+          }}
+          key={item}
+        >
+          <Link to={`www.youtube.com`}>{item}</Link>
```

Comment:
this line is different to the current https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/horizontal-nav-list.js

```suggestion
          <Link to={`${slug.slice(0, -1)}#${item.toLowerCase()}`}>{item}</Link>
```

Reply:
also the `Link` and `mediaQueries` import is a little bit different

Reply:
> this line is different to the current https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/horizontal-nav-list.js

Sorry, I used that for debugging and forgot to change it back. It is fixed now.

> also the `Link` and `mediaQueries` import is a little bit different from the source file

I changed it because I found copying whole `gatsby-plugin-theme-ui` a little bit of an overkill. I changed import for link, because in `gatsby-link` README.md it says not to import from there anymore and use `gatsby` package instead.